### ISSUE TITLE
Change shortcut for Rectangle Paint to avoid keyboard layout switching

### ISSUE THREAD

**Issue by GTcreyon** [godotengine/godot-proposals#3380]
### Describe the project you are working on

A simple top-down fangame of Undertale/FNaF, using tilesets from Undertale.

### Describe the problem or limitation you are having in your project

When using the Rectangle Paint tilemap feature, I often find that I accidentally switch my keyboard layout - the shortcut on Windows for this is Ctrl+Shift.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Change the shortcut to Ctrl+LMB. This only applies to the tilemap brush, because it's used repeatedly, in cycles that can be interrupted.

Usually, shortcuts that use Ctrl+Shift aren't an issue, since the third input of the chord prevents Windows changing the keyboard layout, but personally I often find myself changing my mind halfway through a chord when I'm tilemapping and find out that I've accidentally switched my layout a few minutes later when I try to type.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Simply change the hotkey from Ctrl+Shift+LMB to Ctrl+LMB, or something similar.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

Perhaps with an AutoHotkey script or similar, but this is probably unreasonable.

### Is there a reason why this should be core and not an add-on in the asset library?

Most users don't even know this is an issue, and thus wouldn't know where to look to fix it.

**Comment by Calinou** [comment]
> I often find that I accidentally switch my keyboard layout - the shortcut on Windows for this is Ctrl+Shift.

Isn't it <kbd>Alt + Shift</kbd>?

**Comment by pycbouh** [comment]
> Isn't the keyboard layout switching combination Alt + Shift?

Not sure if it's dependent on something, but in my experience on a fresh install Alt + Shift is locale switching and there is no layout switching shortcut, but many move locale change to Ctrl + Shift and set Alt + Shift for layout switching (I think it even does that automatically).

With all that said, many professional tools that rely on keyboard shortcuts use various combinations of modifier keys. So it's impossible to completely avoid that.

**Comment by GTcreyon** [comment]
> Isn't the keyboard layout switching combination Alt + Shift?
No, not for me at least. It also works with Win+Space, but that's not related to Godot, of course.

**Comment by KoBeWi** [comment]
> Isn't the keyboard layout switching combination Alt + Shift?

It is Ctrl + Shift.

If you don't want to switch layouts, you can just delete the other layout. It's the same problem as with sticky keys. It's just something you always disable on fresh Windows.

As for the shortcut, it's gone in 4.0, but I opened a PR that re-adds it: https://github.com/godotengine/godot/pull/51695
We already have problem with RMB for erasing tiles, I'd be sad if some specific button combo killed another useful shortcut :< We should probably make it configurable, idk.
